cloud: create the TiDB Cloud partner doc (#18785)
pingcap/docs: tidb-cloud/tidb-cloud-partners.md

---
title: TiDB Cloud Partners
summary: Learn how to become an AWS Channel Partner Private Offer (CPPO) partner and Managed Service Provider (MSP) partner.
---

# TiDB Cloud Partners

The TiDB Cloud Partner programs aim to establish and foster a strong partnership between PingCAP and our partners to benefit our customers. 

There are two types of TiDB Cloud partners:

- AWS Channel Partner Private Offer (CPPO)
- Managed Service Provider (MSP)

## AWS Channel Partner Private Offer (CPPO)

[AWS CPPO](https://aws.amazon.com/marketplace/features/cpprivateoffers) allows customers to purchase TiDB Cloud through the AWS Marketplace directly from channel partners. This enables customers to benefit from the partner's business knowledge, localized support, and expertise, while still enjoying the fast and seamless purchasing experience they expect from AWS Marketplace.

### Become an AWS CPPO partner of PingCAP

> If you are interested in the CPPO program and would like to join as a partner, [contact sales](https://www.pingcap.com/partners/become-a-partner/) to enroll.

Author: ```suggestion
If you are interested in the AWS CPPO program and would like to join as a partner, [contact sales](https://www.pingcap.com/partners/become-a-partner/) to enroll.
```

### Manage daily tasks for AWS CPPO customers

> As an AWS CPPO partner, there are two methods for you to manage your daily management tasks:

Author: ```suggestion
As an AWS CPPO partner, you have two ways to manage your daily management tasks:
```

---

> title: TiDB Cloud Partners
> summary: Learn how to become an AWS Channel Partner Private Offer (CPPO) partner and Managed Service Provider (MSP) partner.

Reviewer: ```suggestion
title: TiDB Cloud Partner Web Console
summary: Learn how to become a reseller and Managed Service Provider (MSP).
```

aliases: ['/tidbcloud/managed-service-provider']
---

> # TiDB Cloud Partners

Reviewer: ```suggestion
# TiDB Cloud Partner Web Console
```

> The TiDB Cloud Partner programs aim to establish and foster a strong partnership between PingCAP and our partners to benefit our customers. 
> 
> There are two types of TiDB Cloud partners:
> 
> - AWS Channel Partner Private Offer (CPPO)
> - Managed Service Provider (MSP)

Reviewer: ```suggestion
TiDB Cloud Partner Web Console is designed for partners focused on SaaS solutions, with the goal of building and nurturing a strong partnership between PingCAP and our partners to better serve our customers.

There are two types of TiDB Cloud partners:
- Reseller: Resell TiDB Cloud through AWS Marketplace Channel Partner Private Offer (CPPO)
- Managed Service Provider (MSP)
```

> ## AWS Channel Partner Private Offer (CPPO)

Reviewer: ```suggestion
## Reseller through AWS Channel Partner Private Offer (CPPO)
```

> [AWS CPPO](https://aws.amazon.com/marketplace/features/cpprivateoffers) allows customers to purchase TiDB Cloud through the AWS Marketplace directly from channel partners. This enables customers to benefit from the partner's business knowledge, localized support, and expertise, while still enjoying the fast and seamless purchasing experience they expect from AWS Marketplace.

Reviewer: ```suggestion
Reseller through [AWS CPPO](https://aws.amazon.com/marketplace/features/cpprivateoffers) allows customers to purchase TiDB Cloud through the AWS Marketplace directly from channel partners. This enables customers to benefit from the partner's business knowledge, localized support, and expertise, while still enjoying the fast and seamless purchasing experience they expect from AWS Marketplace.
```

> ### Become an AWS CPPO partner of PingCAP

Reviewer: ```suggestion
### Become a reseller of PingCAP
```

> Reseller through [AWS CPPO](https://aws.amazon.com/marketplace/features/cpprivateoffers) allows customers to purchase TiDB Cloud through the AWS Marketplace directly from channel partners. This enables customers to benefit from the partner's business knowledge, localized support, and expertise, while still enjoying the fast and seamless purchasing experience they expect from AWS Marketplace.

Author: ```suggestion
Reseller through [AWS CPPO](https://aws.amazon.com/marketplace/features/cpprivateoffers) allows customers to purchase TiDB Cloud through the AWS Marketplace directly from a reseller. This enables customers to benefit from the partner's business knowledge, localized support, and expertise, while still enjoying the fast and seamless purchasing experience they expect from AWS Marketplace.
```

> If you are interested in the AWS CPPO program and would like to join as a partner, [contact sales](https://www.pingcap.com/partners/become-a-partner/) to enroll.

Reviewer: ```suggestion
If you are interested in the reseller program and would like to join as a partner, [contact sales](https://www.pingcap.com/partners/become-a-partner/) to enroll.
```

> ### Manage daily tasks for an AWS CPPO

Reviewer: ```suggestion
### Manage daily tasks for a reseller
```

> As an AWS CPPO partner, you have two ways to manage your daily management tasks:

Reviewer: ```suggestion
As a reseller, you have two ways to manage your daily management tasks:
```
